src/Models/Message.php

```diff
@@ -85,7 +86,7 @@ public function conversation()
      *
      * @return Model
      */
-    public function send(Conversation $conversation, string $body, Participation $participant, string $type = 'text', string $data = ''): Model
+    public function send(Conversation $conversation, string $body, Participation $participant, string $type = 'text', array $data): Model
```

Comment:
missing default value for data since optional
`public function send(Conversation $conversation, string $body, Participation $participant, string $type = 'text', array $data = [])`

Reply:
Have just fixed that